routes/definitions.js

```diff
@@ -96,8 +97,15 @@ async function listDefinitions(request, response) {
 
 let definitionService
 
-function setup(definition) {
+function setup(definition, testFlag = false) {
   definitionService = definition
+
+  test = testFlag
```

Comment:
Is there any reason why we need to reassign testFlag to test instead of just using the parameter?

Reply:
Just fixed that :)